packages/patternfly-4/react-core/src/tsconfig.dts.json

```diff
@@ -1,35 +1,23 @@
 {
   "compilerOptions": {
     "outDir": "../dist/js",
-    "allowSyntheticDefaultImports": true,
-    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": false,
```

Comment:
Should we just remove this it will default to esModuleInterop from looking at the typescript documentation I believe.

Reply:
I'll remove it in my next PR which does some typescript groundwork.